### PR TITLE
chore: change some texts in Account info section

### PR DESCRIPTION
**Pull request by mirovladimitrovski**
## Description

Changes outlined in [this Notion card](https://www.notion.so/Viewer-changes-in-web-app-account-info-section-27e600d114064d2dae30ef762c269c41).

image

### Steps completed:

According to our definition of done, I have completed the following steps:

- [ ] Acceptance criteria met
- [ ] Unit tests added
- [ ] Docs updated (including config and env variables)
- [ ] Translations added
- [ ] UX tested
- [ ] Browsers / platforms tested
- [ ] Rebased & ready to merge without conflicts
- [ ] Reviewed own code

### CODE REVIEW

File: src/components/Form/FormSection.tsx [hunk 1]
```diff
@@ -152,13 +152,15 @@ export function FormSection<TData extends GenericFormValues>({
...
-      )}
+      {content ? (
+        isEditing ? (
+          <form className={styles.flexBox} noValidate onSubmit={(event) => event.preventDefault()}>
+            {content({ values, isEditing, isBusy, onChange, errors: formErrors })}
+          </form>
+        ) : (
+          <div className={styles.flexBox}>{content({ values, isEditing, isBusy, onChange })}</div>
+        )
+      ) : null}
```
Comment (budzworthy) [inline src/components/Form/FormSection.tsx]:
```suggestion
      {content && (
        isEditing ? (
          <form className={styles.flexBox} noValidate onSubmit={(event) => event.preventDefault()}>
            {content({ values, isEditing, isBusy, onChange, errors: formErrors })}
          </form>
        ) : (
          <div className={styles.flexBox}>{content({ values, isEditing, isBusy, onChange })}</div>
        )
      )}
```

File: test-e2e/tests/account_test.ts [hunk 2]
```diff
@@ -48,19 +48,18 @@ function runTestSuite(config: typeof testConfigs.svod, providerName: string, res
       I.see(editAccount);
     }
 
-    I.see('Security');
     I.see('Password');
     I.see('****************');
```
Comment (budzworthy) [inline test-e2e/tests/account_test.ts]: This is why your tests are broken:
```suggestion
```

**Review by budzworthy — COMMENTED** [review]
Just a small code style change and then fix the tests by removing the check for '*********'

File: src/i18n/locales/nl_NL/common.json
Comment (budzworthy) [inline src/i18n/locales/nl_NL/common.json:1]: It looks like these didn't merge nicely with the latest translations changes from videodock. Translation strings are now in https://github.com/jwplayer/ott-web-app/tree/develop/public/locales

We also need to make sure that Spanish translations get added for any new strings.

**Review by kmitrovv — APPROVED** [review]
lgtm

**Pull request merged**